pycroft/model/config.py

```diff
@@ -72,6 +72,11 @@ class Config(IntegerIdModel):
         foreign_keys=[membership_fee_bank_account_id]
     )
 
+    non_attributable_transactions_account_id: Mapped[int] = col(ForeignKey(Account.id))
+    non_attributable_transactions_account: Mapped[Account] = relationship(
+        foreign_keys=[non_attributable_transactions_account_id]
+    )
```

Comment:
Remember to add this to the `ConfigFactory`: https://github.com/agdsn/pycroft/blob/865ebcf6bfc5a7a3b02966bf64e00c2342b42717/tests/factories/config.py#L36-L38